ourprojects/models.py

```diff
@@ -16,12 +28,40 @@
 )
 
 
+class Person(Record, CanCurate, TracksRun, TracksUpdates, ValidateFields):
+    """Internal and external people that can be a part of projects or references.
+
+    Example:
+        >>> person = Person(
+        ...     name="Jane Doe",
+        ...     email="jane.doe@example.com",
+        ...     internal=True,
+        ... ).save()
+    """
+
+    class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
+        abstract = False
+
+    id: int = models.AutoField(primary_key=True)
+    """Internal id, valid only in one DB instance."""
+    uid: str = CharField(unique=True, max_length=8, db_index=True, default=ids.base62_8)
+    """Universal id, valid across DB instances."""
+    name: str = CharField(db_index=True)
+    """Name of the person (forename(s) lastname)."""
+    email: str | None = EmailField(null=True, default=None)
+    """Email of the person."""
+    internal: bool = BooleanField(default=False, db_index=True)
```

Comment:
Do we prefer `internal` over `external`?

`external` could be better because it rings the alarm bells. Google Workspace e.g. always highlights with this language: "this person is external, are you sure you want to share this?"

Saying "this person is not internal" is more cumbersome.

We expect the majority of the people in this registry to be internal and auto-synced from the `User` registry. 

For manually created records, we'd like them to default to "external" and it could be more intuitive to then read `Person()` and see: ahh this one is "external" rather than seeing a negated internal. I think there is more awareness when seeing `external=True` rather than `internal=False`. 🤔

Reply:
I agree, thanks. Will change.